cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax.asciidoc

```diff
@@ -1,107 +0,0 @@
-.User management command syntax
```

Comment:
neo4j-documentation/4.3/cypher/cypher-docs/target/docs/dev/ql/administration/security/administration-security-users-and-roles.adoc